brainglobe_atlasapi/atlas_generation/atlas_scripts/princeton_mouse.py

```diff
@@ -127,9 +127,9 @@ def add_parent_id(child_id):
     meshes_dir_path.mkdir(exist_ok=True)
 
     tree = get_structures_tree(structs_dict)
-    rotated_annotations = np.rot90(annotated_volume, axes=(0, 2))
+    # rotated_annotations = np.rot90(annotated_volume, axes=(0, 2))
```

Comment:
```suggestion
```
Let's get rid of this entirely.
From discussions at our last dev meeting this was likely here because of transformation issues in brainrender.